README.md

```diff
@@ -18,17 +18,17 @@ This Project serves the backend APIs required for [Real Dev Squad](https://reald
 
 ## Running the Project
 ```shell
-$ npm install
-$ npm start
+npm install
```

Comment:
The `$` sign signifies a shell command. I dont think we should remove them 🤔

Reply:
I think we should.
Since people are expected to run the commands in their terminal, and everyone's shell prompt may not be a '$'

Mine certainly isn't:
![image](https://user-images.githubusercontent.com/1935403/149441756-77b5e7ef-983b-4ab8-b95f-8ab1fe9695ca.png)